README.md

```diff
@@ -53,39 +57,40 @@ NB: you can also download MeiliSearch from **Homebrew** or **APT**.
 ```ruby
 require 'meilisearch'
 
-client = MeiliSearch::Client.new('url', 'apiKey')
-index = client.create_index(name: 'Books', uid: 'books') # If your index does not exist
-index = client.index('books')                            # If you already created your index
+client = MeiliSearch::Client.new('http://127.0.0.1:7700/', 'masterKey')
+index = client.create_index('books') # If your index does not exist
+index = client.index('books')        # If you already created your index
 
 documents = [
-  { id: 123,  title: 'Pride and Prejudice' },
-  { id: 456,  title: 'Le Petit Prince' },
-  { id: 1,    title: 'Alice In Wonderland' },
-  { id: 1344, title: 'The Hobbit' },
-  { id: 4,    title: 'Harry Potter and the Half-Blood Prince' },
-  { id: 42,   title: 'The Hitchhiker\'s Guide to the Galaxy' }
+  { book_id: 123,  title: 'Pride and Prejudice' },
+  { book_id: 456,  title: 'Le Petit Prince' },
+  { book_id: 1,    title: 'Alice In Wonderland' },
+  { book_id: 1344, title: 'The Hobbit' },
+  { book_id: 4,    title: 'Harry Potter and the Half-Blood Prince' },
+  { book_id: 42,   title: 'The Hitchhiker\'s Guide to the Galaxy' }
 ]
-index.add_documents(documents) # => { "updateId": 1 }
+index.add_documents(documents) # => { "updateId": 0 }
 ```
 
-With the `updateId`, you can check the status of your documents addition thanks to this [method](https://github.com/meilisearch/meilisearch-ruby#update-status).
+With the `updateId`, you can check the status (`processed` or `failed`) of your documents addition thanks to this [method](#update-status).
```

Comment:
You want this to be changed in all SDK's readme?

Reply:
As you want.
I didn't want to annoy you with that because I want to pass completely (soon) on all the readmes, define one template (which will be validate by all of us) and then, change them.